models/users/scratch/snowplow_web_users_this_run.sql

```diff
@@ -23,19 +23,27 @@ select
 
   -- first page fields
   a.first_page_title,
-
   a.first_page_url,
-
   a.first_page_urlscheme,
   a.first_page_urlhost,
   a.first_page_urlpath,
   a.first_page_urlquery,
   a.first_page_urlfragment,
 
-  c.last_page_title,
+  a.geo_country,
+  a.country,
+  a.continent,
+  a.geo_city,
+  a.geo_region_name,
+  a.br_lang,
+  a.language,
```

Comment:
Sorry to be a pain, I think in this table, as we are adding the columns brand new anyway, we could prefix these with `first`...

Reply:
Ah I was thinking, since we're leaving out the `first_` prefix on the sessions model then for consistency's sake we would keep it the same for the users model and note that these columns (the ones without the `last_` prefix) are the first event's fields at the start of a session in the docs, just like what we'll be doing for the sessions model/docs.

Happy to rename these with the prefix otherwise, just my 2 cents.

Reply:
Ah I see, I'll update these cols names with the prefix to make it consistent within this user model.